matrix/count_islands_in_matrix.py

```diff
@@ -0,0 +1,35 @@
+# An island in matrix is a group of linked areas, all having the same value. 
+# This code couts number of islands in a given matrix, with including diagonal connections.
+
+class matrix: # Public class to implement a graph
+ 
+	def __init__(self, row, col, g): # Class constructor
```

Comment:
```suggestion
	def __init__(self, row: int, col: int, graph: list): # Class constructor
```
Python type hints as discussed in CONTRIBUTING.md.